### PR TITLE
Create anny prompts and assets

### PR DESCRIPTION
**Pull request by tioopental**
yes

### CODE REVIEW

File: src/prompts/vedal.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    'tba.mp3'
+  ],
+
+  prompts: [
+    'cringe joke',
+    '"Shut up"',
+    'aruges with Alex',
+    'drunk stream',
+    'argues with Neuro',
+    '"That\'s crazy 🐢"',
```
Comment (tioopental) [inline src/prompts/vedal.js:18]: This one is already in neuro prompts :/

File: src/prompts/anny.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  name: 'anny',
+  color: 'tba',
+
+  image: 'tba.png',
+
+  sounds: [
+    'tba.mp3'
+  ],
+
+  prompts: [
```
Comment (cjmaxik) [inline src/prompts/anny.js:12]: We need at least 24 prompts.

File: src/prompts/vedal.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  name: 'vedal',
+  color: 'tba',
+
+  image: 'tba.png',
+
+  sounds: [
+    'tba.mp3'
+  ],
+
+  prompts: [
```
Comment (cjmaxik) [inline src/prompts/vedal.js:12]: We need at least 24 prompts.

**Comment by cjmaxik** [comment]
We need at least 24 prompts + **tba** fields should be filled.

**Comment by tioopental** [comment]
Adding folders in browser on github physically hurts my brain

File: src/prompts/vedal.js (hunk 1 above)
Comment (cjmaxik) [reply src/prompts/vedal.js:18]: Technically a different occasion

**Comment by cjmaxik** [comment]
Hit `.` on the keyboard and it will open a full editor with dir tree.

**Comment by tioopental** [comment]
anny prompts + assets look fine for now I guess

File: src/prompts/anny.js (hunk 2 above)
Comment (tioopental) [reply src/prompts/anny.js:12]: fixed

File: src/prompts/vedal.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    'code should work, but doesn\'t',
+    'gets hit on by Neuro',
+    'leaks something',
+    
+    // mentions
+    'mentions anny',
+    'mentions programming socks',
+
+    // chat related
+    'chat shoots Tutel',
```
Comment (cjmaxik) [inline src/prompts/vedal.js:28]: Neuro has this one already

```suggestion
```

File: src/prompts/vedal.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    'chat demands attention to Neuro',
+    'chat spams KEKW'
```
Comment (cjmaxik) [inline src/prompts/vedal.js:30]: Neuro has this one already

```suggestion
```

**Review by cjmaxik — CHANGES_REQUESTED** [review]

**Comment by tioopental** [comment]
Fixed

**Comment by cjmaxik** [comment]
TODO for me: update `prompts.js`

**Pull request merged**